maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  display: flex;
+  padding: 0px 0px 0px 50px;
+  align-items: center;
+}
+
+.title img {
+  height: 90px;
+  width: 60px;
+  float: left;  
+}
+
+.title p {
+  font-size: 24px;
+  margin: 0px 0px 0px -15px;
```

Comment:
Dont use negative margins, never! 😅 
You will remove this by using pseudo elements